src/ui/ui.css

```diff
@@ -19,18 +19,44 @@ body {
   margin: 0;
 }
 
+.swg-loading-container {
+  position: absolute !important;
+  width: 100% !important;
+  height: 148px !important;
+  bottom: 0 !important;
+  margin-top: 5px !important;
+  z-index: 2147483647 !important;
+}
+
+/**
+ * Since the desktop view has the parent iframe (.swg-dialog) transperant,
+ * this style adds the background with borders to the loading indicator.
+ */
+@media (min-width: 630px), (min-height: 630px) {
+  .swg-loading-container {
+    width: 560px !important;
+    margin-left: 35px !important;
+    border-top-left-radius: 8px !important;
+    border-top-right-radius: 8px !important;
+    background-color: rgba(255, 255, 255, 1) !important;
+    box-shadow:
+        rgba(60, 64, 67, .3) 0 1px 1px, rgba(60, 64, 67, .15) 0 1px 4px 1px !important;
+  }
+}
+
 swg-container,
 swg-loading,
 swg-loading-animate,
 swg-loading-image {
   display: block;
 }
 
+swg-loading-container,
```

Comment:
Hmm. Here's it's already tag. Pls ensure this is tested.

Reply:
This was a duplicate. Tests are already in place.